help-ja.txt

```diff
@@ -12,13 +12,17 @@ AnkiDroidに限らずPC版Anki（あるいは他のAnkiシリーズ）にもあ
 AnkiDroidに関する質問は、 https://groups.google.com/forum/\#!forum/anki-android[ユーザーフォーラム]（訳注：日本語用のユーザーフォーラムは https://groups.google.com/forum/\#!forum/ankidroid-nihon[こちら]）に投稿するか、フォーラムのmailto:public-forum@ankidroid.org[アドレス]（訳注：日本語用フォーラムのアドレスはmailto:ankidroid-nihon@googlegroups.com[こちら]）にメールしてください。
 
 == バグレポートや新機能リクエスト
-バグ（不具合）の報告や新機能のリクエストを送りたい場合は、まず  https://code.google.com/p/ankidroid/issues/list[AnkiDroidイシュートラッカー]に公開されているイシュー（課題）リストに同じ内容がないかどうか確認してみてください。ない場合は、新しいイシューを作成し、できるだけ詳しい情報を記載してください。バグ報告の場合は、できればエラーログも送信してください。
+バグ（不具合）の報告や新機能のリクエストを送りたい場合は、まず https://code.google.com/p/ankidroid/issues/list[AnkiDroidイシュートラッカー]に公開されているイシュー（課題）リストに同じ内容がないかどうか確認してみてください。
+ない場合は、新しいイシューを作成し、できるだけ詳しい情報を記載してください。
+バグ報告の場合は、下記の手順で「デバッグ情報」も記載してください。これがあると、サポートチームはあなたが書いたレポートとあなたのAnkiDroidが送信したクラッシュレポートとを照らし合わせることができます。
 
-エラーログを送る方法には次の2つがあります。
+「デバック情報」の入手方法
 
- * クラッシュ(アプリの異常終了)を再現した後（訳注：事前にドロワーメニューの[設定]>[AnkiDroid]>[エラーの報告]で、エラーをレポートする設定にしておく必要があります）、 http://ankidroid-triage.appspot.com/report_crashes/report_crashes[AnkiDroid トリアージ]のページを開き、「Filter on version」というプルダウンメニューで、あなたが使用しているAnkiDroidのバージョンを選択してください。選択後のリスト内に、あなたが先ほど送ったクラッシュレポートがあるはずです。そのレポートへのリンクをバグレポートの文中に記載してください。
-
- * AnkiDroid自体でクラッシュレポートを送信できないエラーについては、 https://play.google.com/store/apps/details?id=com.nolanlawson.logcat[Catlog]のようなアプリを使ってエラーログを作成し、バグレポートに添付してください。（注）Android 4.1以降では、こうしたログアプリを使うためにはroot権限が必要です。
+1. 画面左上の星マークのボタンをタップしてドロワーメニューを開いてください。
+2. [設定]をタップしてください。
+3. [AnkiDroid について]をタップしてください。
+4. 画面下の[デバッグ情報をコピー]をタップしてください。（「デバッグ情報」がクリップボードにコピーされます。）
+5. クリップボードの内容をイシューの文中に貼り付けてください。
 
 == AnkiDroidへの貢献
 AnkiDroidはオープンソースプロジェクトですので、このアプリの発展に貢献したいという方であればどなたでも（アプリ開発者でない方も）歓迎いたします。AnkiDroidへの貢献に関心を持たれた方は、最初にAnkiDroid公式サイトの https://code.google.com/p/ankidroid/wiki/Contribution[“Contribution”](貢献)のページ（訳注：英語です）をご覧の上、 https://groups.google.com/forum/\#!forum/anki-android[ユーザーフォーラム]（訳注：日本語用のユーザーフォーラムは https://groups.google.com/forum/\#!forum/ankidroid-nihon[こちら]）にお問い合わせください。
```

Comment:
Yes it seems to be a bug in asciidoctor when two links are together. I'm not sure....
Anyway, I think it's better to link directly to the Japanese forum to be honest, and then we don't have that problem either. I've updated the links ^^^